charts/node-analyzer/README.md

```diff
@@ -154,7 +154,7 @@ The following table lists the configurable parameters of the Sysdig Node Analyze
 | `nodeAnalyzer.hostScanner.additionalDirsToScan`                      | Optional comma-separated list of directories in addition to default ones                 | ` `                                                                                                                                                                   |
 | `nodeAnalyzer.hostScanner.env`                                       | Extra environment variables that will be passed onto pods                                | `{}`                                                                                                                                                                  |
 | `nodeAnalyzer.hostScanner.image.repository`                          | The image repository to pull the Host Scanner from                                       | `sysdig/vuln-host-scanner`                                                                                                                                            |
-| `nodeAnalyzer.hostScanner.image.tag`                                 | The image tag to pull the Host Scanner                                                   | `0.3.2`                                                                                                                                                               |
+| `nodeAnalyzer.hostScanner.image.tag`                                 | The image tag to pull the Host Scanner                                                   | `0.3.4`                                                                                                                                                               |
 | `nodeAnalyzer.hostScanner.image.digest`                              | The image digest to pull                                                                 | ` `                                                                                                                                                                   |
```

Comment:
Should we add to the readme the `nodeAnalyzer.hostScanner.probesPort` parameter?

Reply:
yes please